index.html

```diff
@@ -184,7 +184,52 @@ <h2 class="post-tittle">Multi-Post Stories</h2>
           </div>
         </div>
         </div> 
-      </section>
-    
+    </section>
+
+    <section class="contact-container">
+      <div class="frame42">
+        <h2>CONTACT US</h2>
+        <p>
+          If you have an application you are
+          interested in developing, a feature that you
+          need built or a project that needs coding.
+          I’d love to help with it
+        </p>
+      </div>
+      <form method="post" action="https://formspree.io/f/mjvlplnjl">
```

Comment:
- [ ] Please add some paddings to the form fields and make sure it matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/166517521-4d313622-fd4b-49dc-ba62-aadccaabdb84.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/166517567-badaabcc-e207-4e18-91dd-ccaba081e686.png)